packages/react/src/components/List/_list.scss

```diff
@@ -38,6 +38,7 @@
 
         .#{$iot-prefix}-simple-pagination-page-label {
           flex-grow: 2;
+          text-align: right;
```

Comment:
This goes against our design spec for simple pagination and should not be required to fix your defect.

Reply:
@Ujjwal-UJ lets remove this.

Reply:
removed pagination styles